### DISCUSSION TITLE
Redesign the workflow edit page adding more room for new filters and options

### DISCUSSION THREAD

**Discussion by andergmartins** [publishpress/PublishPress-Planner#1337] [Feature Requests]
We need more room for adding more options for some filters. Having 3 columns limits a lot of what we do in the filters.
I would like to propose a more "vertical" layout. It could be similar to what Zapier and others automate/workflow plugins provide:

![image](https://user-images.githubusercontent.com/407131/108547896-d7d53800-72c9-11eb-9fe5-76a0fda01ae6.png)

Each panel can be like an accordion panel.

**Comment by stevejburge** [comment]
@andergmartins Agree 100%. I had thought to raise this issue too

We should probably move to a 1 column, vertical layout
